pkg/metrics/venafi.go

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright 2022 The cert-manager Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package metrics contains global structures related to metrics collection
+// cert-manager exposes the following metrics:
+// certificate_expiration_timestamp_seconds{name, namespace}
+// certificate_renewal_timestamp_seconds{name, namespace}
+// certificate_ready_status{name, namespace, condition}
+// acme_client_request_count{"scheme", "host", "path", "method", "status"}
+// acme_client_request_duration_seconds{"scheme", "host", "path", "method", "status"}
+// venafi_client_request_duration_seconds{"scheme", "host", "path", "method", "status"}
+// controller_sync_call_count{"controller"}
```

Comment:
This comment is already in metrics.go.  (minus the venafi metric description). Was it accidentally copy-pasted?

Reply:
Thanks, that was me being unsure whether package comments are meant to go in each file and following the convention in this package. I now see that [package comments should only be added in one file](https://go.dev/doc/effective_go#:~:text=For%20multi%2Dfile%20packages%2C%20the%20package%20comment%20only%20needs%20to%20be%20present%20in%20one%20file) so I have removed this from all files in this package except for `metrics.go` and updated it there.